src/main/java/frc/robot/commands/AutonomousDrive.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystems.ChassisSubsystem;
+
+public class AutonomousDrive extends SequentialCommandGroup {
```

Comment:
I would recommend to rename this class to be more specific what the autonomous mode is trying to accomplish. In this case, it only makes a 90 degree turn. What other path does this going to do?